packages/dashboard/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "predev": "docker-compose -f ./docker-compose.dev.yml up --detach --remove-orphans",
-    "dev": "SIDETREE_METHOD='example:sidetree.testnet' next dev",
+    "dev": "dotenv -e .env.development.local next dev",
```

Comment:
given that we now know this works with "ganache"  locally... can we add an example of that as well?